### PR TITLE
Better user documentation

### PR DESCRIPTION
**Pull request by albsch**
Added helpful documentation to use etylizer. Added reference for `-h` when no paths are given.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,22 @@ Static typechecker for Erlang based on set-theoretic types.
 
 ## User-level documentation
 
-Yet to come...
+* `make` will generate a standalone portable escript called `ety` inside the directory
+  `_build/default/bin`
+* `./ety -h` for help
+* To check a single module file `hello.erl` execute `./ety hello.erl`
+* To check a rebar project `hello.erl`
+    * put `ety` inside the root
```
Comment (skogsbaer) [inline README.md:14]: you don't need to do that. You can just `PATH_TO_ETYLIZER/ety`

File: README.md [hunk 2]
```diff
@@ -6,7 +6,22 @@ Static typechecker for Erlang based on set-theoretic types.
 
 ## User-level documentation
 
-Yet to come...
+* `make` will generate a standalone portable escript called `ety` inside the directory
```
Comment (skogsbaer) [inline README.md:9]: or `rebar3 escriptize`

**Review by skogsbaer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (albsch) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (albsch) [reply README.md]: Fixed

**Review by skogsbaer — APPROVED** [review]

**Pull request merged**